vendor/modules.txt

```diff
@@ -56,7 +56,7 @@ github.com/nmiyake/pkg/dirs
 github.com/nmiyake/pkg/errorstringer
 # github.com/nwaples/rardecode v1.0.0
 github.com/nwaples/rardecode
-# github.com/palantir/conjure-go/v6 v6.1.0
+# github.com/palantir/conjure-go/v6 v6.3.0
```

Comment:
Correct way to update is to bump the version in `go.mod` and then run `./godelw mod` (or you can run `./godelw exec -- go get github.com/palantir/conjure-go/v6` and then run `./godelw mod`)